app/views/bids/index.html

```diff
@@ -0,0 +1,29 @@
+<h1>Bids for "<%= @auction.title %>"</h1>
+
+<a href="/auctions/<%= @auction.id %>">« Back to auction</a>
+
+<table class="usa-table-borderless">
+  <thead>
+    <tr>
+      <th scope="col">Bidder</th>
+      <th scope="col">Amount</th>
+      <th scope="col">Date (Eastern Standard Time)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @auction.bids.sort_by {|b| b.created_at}.reverse.each_with_index do |bid, i| %>
```

Comment:
What do you think about adding a scope / class method to `bids` called something like `most_recently_created_first` and using that instead of `@auction.bids.sort_by {|b| b.created_at}` -- that would be easier to read and you could also have a unit test for it!